### ISSUE TITLE
The cuda problem is not being solved

### ISSUE THREAD

**Issue by ieya114** [Linaqruf/kohya-trainer#321]
CUDA backend failed to initialize: Found cuDNN version 8700, but JAX was built against version 8904, which is newer. The copy of cuDNN that is installed must be at least as new as the version against which JAX was built. (Set TF_CPP_MIN_LOG_LEVEL=0 and rerun for more info.)

Nothing is working with this error. How can I solve this problem?

**Comment by chrsgcknhmr** [comment]
I ran TheLastBens sd auto1111 colab and also since Friday I get the following error message:

WARNING[XFORMERS]:
xFormers can't load C++/CUDA extensions. xFormers was built for:
PyTorch 2.1.0+cu118 with CUDA 1106 (you have 2.1.0+cu121)
Python 3.9.16 (you have 3.10.12) Please reinstall xformers (see https://github.com/facebookresearch/xformers#installing-xformers)
Memory-efficient attention, SwiGLU, sparse and more won't be available.
Set XFORMERS_MORE_DETAILS=1 for more details

Is it within Colab/ notebook version issues or is it in someway connected to my local versions? Because which python --version says I'm running on 3.12.

**Comment by ClaraSanders** [comment]
Yeah, as I said above almost every colab related to SD is broken due to Google latest update. There is a temporary fix, I don't know if it will work for Kohya. Check out my topic on The Last Ben's discussion group.

**Comment by chrsgcknhmr** [comment]
Good to know — thank you! What’s/ where is TheLastBens Discussion group?

**Comment by Linaqruf** [comment]
The cuda fix is pushed, it may still showing cuda is not initialized or something but I finished a LoRA training without error, the problem actually in bitsandbytes version.

**Comment by PlumButa** [comment]
> The cuda fix is pushed, it may still showing cuda is not initialized or something but I finished a LoRA training without error, the problem actually in bitsandbytes version.

It can works, thanks for your efforts.
Your trainer has the best training quality among other Colab notes.

**Comment by xjdeng** [comment]
I'm able to train using DAdaption but not Adam8bit as a result of this.
